mne_bids/copyfiles.py

```diff
@@ -153,6 +154,77 @@ def copyfile_ctf(src, dest):
                   op.join(dest, bids_folder_name + ext))
 
 
+def copyfile_kit(src, dest, subject_id, session_id,
+                 task, run, _init_kwargs, make_bids_basename):
```

Comment:
next step is to write a test!

Reply:
I am not quite sure how to write a test, any suggestions?

Reply:
see [here](https://github.com/mne-tools/mne-bids/blob/master/mne_bids/tests/test_copyfiles.py) for inspiration. You will write a new function and then inside it assert that the files have indeed been copied.